de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/InitializationCallbacks.java

```diff
@@ -39,7 +41,7 @@
         private final DesktopPresenterImpl presenter;
         private final UserSessionServiceFacade userSessionService;
         private final UserPreferencesCallback userPreferencesCallback;
-        Logger LOG = Logger.getLogger(BootstrapCallback.class.getName());
+        Logger LOG = Logger.getLogger("BootstrapCallback");
```

Comment:
Stroot and I had a talk about this - we were doing that, but then the names become obfuscated and the obfuscated name is what ends up here which isn't helpful.